src/sonic-pit/pit-sysdiag/src/psu_tc.py

```diff
@@ -0,0 +1,348 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+__version__ = 'v0.1.0'
+__author__  = "Pytool Lishoulei<admin@pytool.com>"
+import sys
+#import json
```

Comment:
please remove All the commented-out lines if not needed.

Reply:
Removed.